OpenKh.Tools.ModsManager/ViewModels/MainViewModel.cs

```diff
@@ -401,6 +424,151 @@ private void MoveSelectedModUp()
             ModEnableStateChanged();
         }
 
+        private async Task PatchGame(bool fastMode)
+        {
+            await Task.Run(() =>
+            {
+                if (ConfigurationService.GameEdition == 2)
```

Comment:
Do we have constants defined anywhere for the game editions? If not, should we?

Reply:
I'd rather not fix what works.

Reply:
Not suggesting it's broken, but magic numbers and strings in a codebase make it really tough for newcomers to figure out what means what. I'm not a core maintainer of this project though, so it's just a suggestion from me.

Reply:
Looks like the precedent in the codebase is to just use the magic numbers for this. Consider an overall suggestion for future if anyone ever wants to improve the readability for newcomers in the future.

Reply:
An enumeration in the future would be nice so mistakes are less common.